### ISSUE TITLE
keys garbled

### ISSUE THREAD

**Issue by pandalin** [redis/RedisDesktopManager#3678]
RDM Version: 0.8.7.322
Environment (OS name and version): Windows 7
Redis Server Version: 3.0

Steps to reproduce:

 keys garbled,but keys on others computer not

**Comment by timotyweis** [comment]
I'm not sure if the issue could be considered related, but I've experienced some strange keys visualizations, probably due to the pattern used for the split. In order to reproduce the problem, here are some keys to add:

```
set x.y.z.ProductData:Products:P.1 ""
set x.y.z.ProductData:Products:P.2 ""
set x.y.z.ProductData:Product:P.1 ""
set x.y.z.ProductData:Product:P.2 ""
```

The keys containing **Products** are correctly grouped together, whereas the one with **Product** are displayed singularly as unrelated keys.

**Comment by uglide** [comment]
Duplicate of https://github.com/uglide/RedisDesktopManager/issues/3666
This issue was fixed in 0.8.8-beta
